### PR TITLE
Mention how to find the root folder

### PR DESCRIPTION
**Pull request by za3k**
Preview: https://github.com/za3k/multimc-wiki/blob/master/Directory-Structure.md

### CODE REVIEW

File: Directory-Structure.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Not all files/directories might be available, depending on your setup
 
-## Linux
+The easiest way to see the root folder is to click `Folders` > `View Instance Folder`. Then, go up one level.
+
+MultiMC stores its files in the same location as the MultiMC application.
```
Comment (Zetabite) [inline Directory-Structure.md]: leave out the its. Makes the sentence shorter and no worries about a missing ' in a word.

File: Directory-Structure.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Not all files/directories might be available, depending on your setup
 
-## Linux
+The easiest way to see the root folder is to click `Folders` > `View Instance Folder`. Then, go up one level.
```
Comment (Zetabite) [inline Directory-Structure.md]: Many people don't understand what is meant with one level. Maybe say parent folder or something. Ideally would be images ofc

File: Directory-Structure.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+
+MultiMC stores its files in the same location as the MultiMC application.
+
+- On Windows, this folder should be wherever you extracted the MultiMC zip file.
+- On Linux and OS X, the default install location is `~/.config/local/multimc`
```
Comment (Zetabite) [inline Directory-Structure.md]: This goes for several lines, i don't think you need most of the commas in proper english

**Comment by za3k** [comment]
@Zetabite Github has a feature where I believe you should be able to make changes to my PR yourself because I've given you permissions. For next time around.

The other small point I'll mention is that the docs say "Directory" consistently, but MultiMC itself says "Folder". I assume people know they're the same?

**Comment by Zetabite** [comment]
> The other small point I'll mention is that the docs say "Directory" consistently, but MultiMC itself says "Folder". I assume people know they're the same?

Always in for consistency

**Comment by za3k** [comment]
> Always in for consistency

Sorry I'm not sure what this means.

**Comment by Zetabite** [comment]
You are welcome to make it more consistent

**Comment by za3k** [comment]
Done. Especially note that I changed the filename, in case any links need updated. Feel free to change the filename back yourself if you don't want to break things.

**Pull request merged**